public/index.html

```diff
@@ -1,14 +1,12 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
+
+<head>
```

Comment:
Do you have prettier auto-formatting properly set up in VSCode?

Reply:
No i don't believe so, I'm looking into right now.

Reply:
It's part of the [Onboarding.md document](https://github.com/Northeastern-Electric-Racing/PM-Dashboard-v2/blob/main/docs/Onboarding.md#ide-vscode), I'd highly recommend checking it out.